src/main/java/org/mockito/internal/util/ObjectMethodsGuru.java

```diff
@@ -17,12 +17,12 @@ public static boolean isToStringMethod(Method method) {
         MockitoMethod m = new DelegatingMethod(method);
         return m.getReturnType() == String.class &&
                m.getParameterTypes().length == 0 &&
-               m.getName().equals("toString");
+               "toString".equals(m.getName());
```

Comment:
@TimvdLippe in this case the flip add no value since the Method.getName() never returns null, it makes the code less readable